src/rules/mod.rs

```diff
@@ -6,6 +6,7 @@ pub(crate) mod flake8_blind_except;
 pub mod flake8_boolean_trap;
 pub mod flake8_bugbear;
 pub(crate) mod flake8_builtins;
+pub mod flake8_commas;
```

Comment:
Can we make this `pub(crate)` for now? (The `pub mod` pieces here have to expose `settings` as-is.)

Reply:
Since #1874 which made `ruff::rules` private all of the modules in `src/rules/mod.rs` should be defined with `pub mod` since `pub(crate) mod` would be redundant.